docs/Dockerfile

```diff
@@ -34,4 +34,4 @@ COPY tox.ini .
 COPY docs/requirements.txt docs/
 
 RUN tox --workdir /tmp/tox --notest -e docs && \
-    rm -rf ~/.cache/pip
+    rm -rf ~/.cache/pip /tmp/tox
```

Comment:
If you remove the workdir, the whole `RUN` instruction here is pointless (it's used to pre-install the required venv)

Reply:
Hmm ok I see, then I will just change the ownership on the directory